libs/core/src/components/sage-avatar/sage-avatar.tsx

```diff
@@ -0,0 +1,62 @@
+import { Component, Host, h, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'sage-avatar',
+  styleUrl: 'sage-avatar.scss',
+  shadow: true,
+})
+export class SageAvatar {
+
+  /**
+   * Determines whether the badge is visible or not
+   * @defaultValue false
+   */
+  @Prop() badge? = false;
+
+  /**
+   * The src for a custom user image.
+   * @defaultValue null
+   */
+  @Prop() image?: string;
+
+  /**
+   * Preset sizes for the avatar. If a custom size is desired,
+   * use the `--size` custom property instead.
+   * @defaultValue md
+   */
+  @Prop() size?: 
+  | 'xl'
+  | 'lg'
+  | 'md' 
+  | 'sm' 
+  | 'xs';
+
+  /**
+   * Determines the variant of avatar. Changes appearance accordingly.
+   * @defaultValue customer
+   */
+  @Prop({ reflect: true }) variant?: 'customer' | 'admin' = 'customer'
+
+  private classNames = (props: object) => {
+    return Object.keys(props).join(' ');
+  }
+
+  render() {
+    return (
+      <Host
+        class={this.classNames({
+          'sage-avatar': true,
+          ...(this.size && { [`sage-avatar--${this.size}`] : []}),
+          ...(this.variant === 'admin' && { [`sage-avatar--${this.variant}`] : []}),
+        })}
+      >
+        <div>
+          {this.image ? <img alt='Profile' src={this.image} /> : <sage-icon name="user-filled" size="normal"></sage-icon>}
```

Comment:
In this case, I'm not entirely sure. I can make a prop for this, no problem. I'm wondering if an avatar's use case will be so different that it will warrant needing a new alt? Any thoughts here? 🤔

Reply:
Yeah, I was a bit on the fence about it so I was just curious to see what you thought. I can see it being beneficial for something like a contacts list where there could be quite a few different photos. I don't think it's a showstopper for now.